pkg/abi/nvgpu/classes.go

```diff
@@ -333,6 +334,49 @@ type NV00F8_ALLOCATION_PARAMETERS struct {
 	Map        nv00f8Map
 }
 
+// From src/common/sdk/nvidia/inc/class/cl00e0.h
+const (
+	NV_MEM_EXPORT_UUID_LEN = 16
+)
+
+// NV_EXPORT_MEM_PACKET is from
+// src/common/sdk/nvidia/inc/class/cl00e0.h
+//
+// +marshal
+type NV_EXPORT_MEM_PACKET struct {
+	UUID   [NV_MEM_EXPORT_UUID_LEN]uint8
+	Opaque [16]uint8
+}
+
+// NV00FD_ALLOCATION_PARAMETERS is the alloc param type for NV_MEMORY_MULTICAST_FABRIC
+// from src/common/sdk/nvidia/inc/class/cl00fd.h
+//
+// +marshal
+type NV00FD_ALLOCATION_PARAMETERS struct {
+	ExpPacket  NV_EXPORT_MEM_PACKET
+	Index      uint16
+	_          [6]byte
```

Comment:
IIUC, the pre-545 version of this struct lacks these fields.

Reply:
Ah good catch, I think `NV00FD_ALLOCATION_PARAMETERS_V545` aims to cover this fact but it wasn't removed from here after versioning support was added.

Reply:
Yep thanks for catching!